tests/testing-framework/src/runtimes/in_process_spin.rs

```diff
@@ -52,9 +52,52 @@ impl InProcessSpin {
 }
 
 impl Runtime for InProcessSpin {
-    type Config = ();
-
     fn error(&mut self) -> anyhow::Result<()> {
         Ok(())
     }
 }
+
+impl InProcessSpin {
```

Comment:
Why not move this `config` method under the impl for `InProcessSpin` a few lines up?